e2e/fn_test.go

```diff
@@ -51,26 +60,34 @@ func runAllTests(t *testing.T, path string) {
 }
 
 func runTests(t *testing.T, cases *runner.TestCases, sequential bool) {
-	for _, c := range *cases {
-		c := c // capture range variable
-		if c.Config.Sequential != sequential {
-			continue
-		}
-		t.Run(c.Path, func(t *testing.T) {
-			if !c.Config.Sequential {
-				t.Parallel()
-			}
-			r, err := runner.NewRunner(t, c, c.Config.TestType)
-			if err != nil {
-				t.Fatalf("failed to create test runner: %s", err)
-			}
-			if r.Skip() {
-				t.Skip()
-			}
-			err = r.Run()
-			if err != nil {
-				t.Fatalf("failed when running test: %s", err)
+	runtime := strings.ToLower(os.Getenv(fnRuntimeEnv))
```

Comment:
may be the environment variable should take list of runtimes (comma separated) instead of just one.

I can override the list if I want to as developer (and test runner can construct one by checking what all runtimes are available by default).

Reply:
Changed it to using env var